plugin/federation/federation.gotpl

```diff
@@ -103,10 +103,10 @@ func (ec *executionContext) __resolve_entities(ctx context.Context, representati
 						if err != nil {
 							return fmt.Errorf(`resolving Entity "{{$entity.Def.Name}}": %w`, err)
 						}
-						{{ range $entity.Requires }}
-							entity.{{.Field.JoinGo `.`}}, err = ec.{{.Type.UnmarshalFunc}}(ctx, rep["{{.Field.Join `"].(map[string]interface{})["`}}"])
+						{{ if $entity.Requires }}
+							err = ec.Populate{{$entity.Def.Name}}Requires(ctx, entity, rep)
```

Comment:
This results in:
`PopulatePlanetRequiresRequires`

Reply:
I don't think so.  You might be thinking of Popluator.FuncName.

Reply:
Pretty sure this template is what winds up causing https://github.com/99designs/gqlgen/pull/2676/files#diff-8574d02e76a5404ff6b01c08b88ce6e4a9f4e381e21de4238a418514e6db1443R31Z
```
func (ec *executionContext) PopulatePlanetRequiresRequires(ctx context.Context, entity *model.PlanetRequires, reps map[string]interface{}
```
And I would prefer to not have the `Requires` repeated as `RequiresRequires`.

Reply:
The model is just named something strange for the test.  Requires is only added once. In the real world the model (the struct) wouldn't be named something like this.